README.md

```diff
@@ -22,20 +22,22 @@ First, make sure to [have Python installed](https://docs.python.org/3/using/inde
 Windows users are recommended to grab the official distribution from Windows Store.
 Yakut requires **Python 3.7 or newer**.
 
+Then, if you are using GNU/Linux, make sure to install these:
+```bash
+sudo apt install libjack-jackd2-dev libasound2-dev can-utils
+```
```

Comment:
Not all distributions use APT. For instance, on my computer, this won't work. This is why we need to keep the original, abstract description (usually, users of GNU/Linux know what they are doing), which can be accompanied by specific instructions for Debian-based distros.